tools/fileconv/api3/models.go

```diff
@@ -72,6 +72,7 @@ func (p PathItem) RetrieveSchemaOperation(
 	locator ObjectArrayLocator,
 	displayProcessor DisplayNameProcessor,
 	operationMethodFilter ReadOperationMethodFilter,
+	propertyShouldFlatten NestedFlattening,
```

Comment:
Can you please add comments to explain all the properties in this `RetrieveSchemaOperation` function?

Reply:
Also this parameter is confusingly named, usually parameters that have "should" in the name are booleans, whereas this is a function. Can you name this parameter `propertyFlattener` and the type `propertyFlattener`?